mbuild/tests/test_utils.py

```diff
@@ -110,3 +111,11 @@ def test_js_utils(self):
                     });
                 """
             ]
+
+    def test_coord_wrap(self):
+        xyz = np.array([[3, 3, 1],
+                        [1, 1, 0]])
+        box = [2,2,2]
+        new_xyz = wrap_coords(xyz, box)
+        (xyz[1,:] == new_xyz[1,:]).all()
+        (new_xyz == np.array([1,1,1])).all()
```

Comment:
How does this pass? Shouldn't new_xyz be `[[1, 1, 1], [1, 1, 0]]`?


```python3
>>> (np.array([[1, 1, 1], [1, 1, 0]]) == np.array([1, 1, 1])).all()
False
```

Reply:
Good catch, it probably doesn't help I forgot `assert`